faker/providers/person/ar_AA/__init__.py

```diff
@@ -1,20 +1,23 @@
 # coding=utf-8
 from __future__ import unicode_literals
+from collections import OrderedDict
+
 from .. import Provider as PersonProvider
 
 
 class Provider(PersonProvider):
-    formats_female = (
-        '{{first_name_female}} {{last_name}}',
-        '{{prefix_female}} {{first_name_female}} {{last_name}}',
-    )
+    formats_female = OrderedDict((
+        ('{{first_name_female}} {{last_name}}', 0.98),
+        ('{{prefix_female}} {{first_name_female}} {{last_name}}', 0.02),
```

Comment:
is there any statistical basis for the assumption that 2% of people will use a prefix?

Reply:
No, unfortunately there were none. However, since `formats_female`, `formats_male` and `formats` are all used in their corresponding _name_ functions, that is `name_male`, `name_female` and `name`, it is assumed that a person's name, most of the time, indicates his/her first and last names only, without the prefix nor the suffix (if available). Also, similar implementation was found in _en_US_ person provider.

Reply:
Our [policy](https://github.com/joke2k/faker/blob/master/docs/coding_style.rst#data-sets) is to introduce new data only based on statistical evidence. Some provider don't have a source for the distribution of prefixes because they didn't have it before the introduction of this policy.

I would love to have default distributions of names vs names and prefix, but I don't know what the most common distribution is among countries. Just assuming it's like the `en_US` seems a little too biased.

Reply:
Fair point. Should I revert that commit then?

Reply:
Yeah. I'm sorry, I wish there was better data available.